releasenotes/template.yaml

```diff
@@ -1,3 +1,5 @@
+apiVersion: releaseNotes/v1
```

Comment:
nit: if we want to follow k8s schema apiVersion cannot have capital letters I am pretty sure

Reply:
For consistency, changing to lowercase:
```
Kubernetes resources can have names up to 253 characters long. The characters allowed in names are: digits (0-9), lower case letters (a-z), -, and ..
```

Not a Kubernetes API so it shouldn't matter, but lets keep it consistent.